hippunfold/workflow/rules/autotop.smk

```diff
@@ -33,6 +33,31 @@ rule laplace_coords:
     log: bids(root='logs',**config['subj_wildcards'],dir='{dir}',hemi='{hemi,Lflip|R}',modality='{modality}',suffix='laplace.txt')
     script: '{params.cmd}'
 
+rule prep_isovolume_coords:
+    input: get_labels_for_laplace,
+    params:
+        src_labels = lambda wildcards: config['laplace_labels'][wildcards.dir]['src'],
+    output:
+        outerbin = bids(root='work',datatype='seg_{modality}',dir='{dir}',suffix='outerbin.nii.gz',space='corobl',hemi='{hemi,Lflip|R}', **config['subj_wildcards']),
+        innerbin = bids(root='work',datatype='seg_{modality}',dir='{dir}',suffix='innerbin.nii.gz',space='corobl',hemi='{hemi,Lflip|R}', **config['subj_wildcards']),
+    log: bids(root='logs',**config['subj_wildcards'],dir='{dir}',hemi='{hemi,Lflip|R}',modality='{modality}',suffix='binarize.txt')
+    script: '../scripts/prep_isovolume_coords.py'
+
+rule isovolume_coords:
+    input: 
+        outerbin = bids(root='work',datatype='seg_{modality}',dir='{dir}',suffix='outerbin.nii.gz',space='corobl',hemi='{hemi,Lflip|R}', **config['subj_wildcards']),
+        innerbin = bids(root='work',datatype='seg_{modality}',dir='{dir}',suffix='innerbin.nii.gz',space='corobl',hemi='{hemi,Lflip|R}', **config['subj_wildcards']),
+    params:
+        src_labels = lambda wildcards: config['laplace_labels'][wildcards.dir]['src'],
+    output:
+        coords = bids(root='work',datatype='seg_{modality}',dir='{dir}',suffix='isovol.nii.gz',space='corobl',hemi='{hemi,Lflip|R}', **config['subj_wildcards']),
```

Comment:
since isovol coords are still coords, I think we should rename to be: desc='isovol', suffix='coords.nii.gz'

then the laplace coords can be renameds to: desc='laplace', suffix='coords.nii.gz'

Reply:
fixed